### PR TITLE
Add documentation for `/api/v2_alpha/notifications`

### PR DESCRIPTION
**Pull request by ClearlyClaire**
Some examples are missing but most of it should be there.

### CODE REVIEW

File: content/en/methods/notifications_alpha.md [hunk 1]
```diff
@@ -0,0 +1,598 @@
...
+##### Query parameters
+
+max_id
+: String. All results returned will be lesser than this ID. In effect, sets an upper bound on results.
+
+since_id
+: String. All results returned will be greater than this ID. In effect, sets a lower bound on results.
+
+min_id
+: String. Returns results immediately newer than this ID. In effect, sets a cursor at this ID and paginates forward.
```
Comment (oneiros) [inline content/en/methods/notifications_alpha.md:63]: What is `ID` in this context? Group keys?

Comment (ClearlyClaire) [reply content/en/methods/notifications_alpha.md:63]: No, they are notification IDs! I will make that explicit.

Comment (ClearlyClaire) [inline content/en/methods/notifications_alpha.md:63]:
```suggestion
max_id
: String. All results returned will be about notifications strictly older than this notification ID. In effect, sets an upper bound on results.

since_id
: String. All results returned will be about notifications strictly newer than this notification ID. In effect, sets a lower bound on results.

min_id
: String. Returns results about notifications immediately newer than this notification ID. In effect, sets a cursor at this ID and paginates forward.
```

Comment (ClearlyClaire) [reply content/en/methods/notifications_alpha.md]: Updated, hopefully this is clearer now!

Comment (oneiros) [reply content/en/methods/notifications_alpha.md]: Yes!

**Review by oneiros — APPROVED** [review]

**Pull request merged**